storage-shared-benchmarking/src/main/java/com/google/cloud/storage/benchmarking/StorageSharedBenchmarkingCli.java

```diff
@@ -80,16 +80,28 @@ public final class StorageSharedBenchmarkingCli implements Runnable {
       description = "Specify the path where the temporary directory should be located")
   String tempDirLocation;
 
-  @Option(names = "-warmup", description = "Warmup in seconds", defaultValue = "0")
+  @Option(
+      names = "-warmup",
+      description = "Number of seconds a W1R3 warmup will run on all available processors",
```

Comment:
Let's keep the number of processors consistent between this and Go. We could either
1. Hard code 4 for both
2. Make it an option to specify the number
3. Change Go to use numCPU as well/

@BrennaEpp what do you prefer?

Reply:
So Go does use numCPU for warmups now, this would be consistent with that. Option 2 would be fine as well as we could make it consistent. I think numCPU is fine for warmups.

Reply:
Ahh okay, yeah in that case we are fine.